readme.md

```diff
@@ -228,6 +193,42 @@ const [useStore] = create(log(immer(set => ({
 }))))
 ```
 
+## Can't live without redux-like reducers and action types?
+
+```jsx
+const types = { increase: "INCREASE", decrease: "DECREASE" }
+
+const reducer = (state, { type, by = 1 }) => {
+  switch (type) {
+    case types.increase: return { count: state.count + by }
+    case types.decrease: return { count: state.count - by }
+  }
+}
+
+const [useStore] = create(set => ({
+  count: 0,
+  dispatch: args => set(state => reducer(state, args)),
+}))
+
+const dispatch = useStore(state => state.dispatch)
+dispatch({ type: types.increase, by: 2 })
+```
+
+Or, just use our redux-middleware. It wires up your main-reducer, sets initial state, and adds a dispatch function to the state itself and the vanilla api. Try [this](https://codesandbox.io/s/amazing-kepler-swxol) example.
+
+```jsx
+import { redux } from 'zustand/middleware'
+
+const [useStore] = create(redux(reducer, initialState))
+```
+
 ## Devtools
 
-Yes, it's currently [being hashed out](https://github.com/react-spring/zustand/issues/6) but you can already start using it: https://codesandbox.io/s/amazing-kepler-swxol. It works with regular actions as well, you don't need reducers for this.
+```jsx
+import { devtools } from 'zustand/middleware'
+
+// Usage with a plain action store, it will log actions as "setState"
```

Comment:
Should we specify it actually logs in the Redux dev tools?

Reply:
i will rename the heading "redux devtools" just in case